src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,10 @@
+package wolox.training.repositories;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import wolox.training.models.Book;
+
+@Repository
+public interface BookRepository extends CrudRepository<Book, Long> {
+    Book findByAuthor(String authorName);
```

Comment:
It's better to use `Optional` here, it will be easier and less prone to error to use this method on the top layers :) 
You can read more about it [here](https://www.baeldung.com/java-optional)

Reply:
Awesome! I've added the Optional class